bridge/src/main/java/com/bridge/processinputhandler/AEventManager.java

```diff
@@ -0,0 +1,17 @@
+package com.bridge.processinputhandler;
+
```

Comment:
src/main/java/com/bridge/processinputhandler/AEventManager.java
          @@ -1,7 +1,6 @@
           package·com.bridge.processinputhandler;
           
           import·com.bridge.ipc.IEventBuffer;
          -
           import·java.util.ArrayList;
           import·java.util.List;
           import·java.util.concurrent.ConcurrentLinkedQueue;
Doesnt acomlish with format, review with ./gradlew build

Run './gradlew :bridge:spotlessApply'

Reply:
There are more archieves that needs a format fix such us:
src/test/java/com/bridge/ipc/ReceiverTest.java
src/test/java/com/bridge/processinputhandler/AEventManagerTest.java
src/test/java/com/bridge/processinputhandler/InputVerifierTest.java
src/test/java/com/bridge/GameTest.java
src/test/java/com/bridge/EventGenerator.java
src/main/java/com/bridge/processinputhandler/IPublisher.java
can be fixed using : **./gradlew :bridge:spotlessApply**